hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -296,7 +319,10 @@ public boolean removeProperties(Namespace namespace,  Set<String> properties) {
     properties.forEach(key -> parameter.put(key, null));
     Database database = convertToDatabase(namespace, parameter);
 
-    return alterHiveDataBase(namespace, database);
+    boolean result = alterHiveDataBase(namespace, database);
+    LOG.debug("Remove properties: {} from: {} resulted: {}", properties, namespace, result);
```

Comment:
Same here. I'd rather not add debug logs for properties.

Reply:
We do not log the actual values, just the properties we are removing.
Could it also contain sensitive information?
If not I would keep the log message, so we can be able to trace things in case of trouble

Reply:
If not logging the properties, then what is the value of this log? If the query fails, it will throw an exception so the only time this will run is when the result is `true`. That doesn't seem useful to me, considering this is called from an operation that probably logs itself, like `updateProperties`.

Reply:
Removed the alterHiveDataBase method return value, since it is a private function and as you correctly pointed out never returns false.
Tried to find the usages of the HiveCatalog.setProperties/removeProperties method, and followed the call stack a few steps, but did not find any logs around that. (I might missed something)